test/util-test.js

```diff
@@ -261,6 +276,65 @@ describe('util.getVideoDescription()', function() {
 });
 
 
+describe('util.getAuthor()', function() {
+  it('Retrieves formatted video author', function() {
+    var html = fs.readFileSync(path.resolve(__dirname,
+      'files/util/related-video'), 'utf8');
+    var authorObj = util.getAuthor(html);
+    assert.deepEqual(authorObj,{
+      type: 'channel',
+      ref: '/channel/UC_aEa8K-EOJ3D6gOs7HcyNg',
+      id: 'UC_aEa8K-EOJ3D6gOs7HcyNg',
+      name: 'NoCopyrightSounds',
+      avatar: 'hisprofile.pic',
+      user: '/user/NoCopyrightSounds',
+	});
+  });
+})
+
+
+describe('util.getPublished()', function() {
+  it('Retrieves formatted published date', function() {
+    var html = fs.readFileSync(path.resolve(__dirname,
+      'files/util/related-video'), 'utf8');
+    var publishedTimestamp = util.getPublished(html);
+    assert.equal(publishedTimestamp.toString(),'1416355200000');
```

Comment:
Space after `,`

You can compare the ints directly without converting to string.